src/invidious/videos.cr

```diff
@@ -1037,7 +1047,9 @@ def extract_video_info(video_id : String, proxy_region : String? = nil, context_
 
   author_info = video_secondary_renderer.try &.dig?("owner", "videoOwnerRenderer")
   author_thumbnail = author_info.try &.dig?("thumbnail", "thumbnails", 0, "url")
+  author_verified_badge = author_info.try &.["badges"]?
```

Comment:
Same as above: you forgot to check if the badges element is an array, and if it has a "verified" badge child!